src/lib/components/utils/acm-tagfield.svelte

```diff
@@ -0,0 +1,109 @@
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte';
+
+  export let tags: string[] = [];
+  export let selected: string[] = [];
+  export let label = '';
+  export let resetButton = '';
+
+  const dispatch = createEventDispatcher();
+  let hasSelectedTags = selected.length > 0;
+
+  function selectTag(event: any) {
```

Comment:
For this warning, I would replace the `any` argument type with the following argument type instead:

```ts
MouseEvent & {
    currentTarget: EventTarget & HTMLDivElement;
}
```